frontend/src/components/Navbar.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import {
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  NavLink,
+  NavbarText
+} from 'reactstrap';
+
+const NavBar = (props) => {
```

Comment:
Can you change this into a functional component instead of a const please. [Here](https://imgur.com/CO0B4U6) is an example of how to do so.